examples/with-mux-video/components/layout.js

```diff
@@ -6,7 +6,7 @@ export default function Layout({
   description,
   metaTitle,
   metaDescription,
-  image,
+  image = '/mux-nextjs-og-image.png',
```

Comment:
I think that `1.27mb` is too much for an og:image

Reply:
optimized and resized down from 2400px to 1200px so now we're looking at ~450 kb